src/pages/api/forks.js

```diff
@@ -29,33 +30,40 @@ Users.hasMany(Users_repositories, {
 });
 
 //function for return forked repo
-const forkedRepos = async (repoId) => {
-  const data = await Repositories.findAll({
-    where: { parent_repo_id: repoId },
-    include: [
-      {
-        model: Repositories,
-        as: "parent",
-        include: [
-          {
-            model: Repositories,
-            as: "children",
+const forkedRepos = async (repoId, res) => {
```

Comment:
why you need to pass res object here? You can simply catch exception in your calling function and send response from there only.